packages/escalade/emulation/emulationstation/config/es_systems-rpi.cfg

```diff
@@ -97,6 +97,14 @@ All systems must be contained within the <systemList> tag.-->
 		<platform>gba</platform>
 	</system>
 	<system>
+		<name>saturn</name>
+		<fullname>Sega Saturn</fullname>
+		<path>/storage/roms/saturn</path>
+		<extension>.bin .cue .iso</extension>
+		<command>/usr/bin/retroarch.start -L /tmp/cores/yabause_libretro.so %ROM%</command>
```

Comment:
I've also added Yabause to RPI EmulationStation here, since it wasn't there before.